codalab/worker_manager/azure_batch_worker_manager.py

```diff
@@ -0,0 +1,157 @@
+try:
+    from azure.batch.batch_auth import SharedKeyCredentials  # type: ignore
+    from azure.batch._batch_service_client import BatchServiceClient  # type: ignore
+    from azure.batch.models import (  # type: ignore
+        OutputFile,
+        OutputFileBlobContainerDestination,
+        OutputFileDestination,
+        OutputFileUploadCondition,
+        OutputFileUploadOptions,
+        TaskAddParameter,
+        TaskCounts,
+        TaskContainerSettings,
+    )
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "Running the worker manager requires the azure-batch module.\n"
+        "Please run: pip install azure-batch==9.0.0"
+    )
+
+import logging
+import os
+import uuid
+from argparse import ArgumentParser
```

Comment:
alphabetical order? not sure if the linter complains about this.

Reply:
I don't think Black complains about this. I actually wasn't sure what the best practice for import statements was, so I followed the style here: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement